cmd/run.go

```diff
@@ -8,6 +8,10 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/manager/signals"
 )
 
+func init() {
+	evalCmd.Flags().Int("loglvl", 1, "Sets verbosity level of combo CR controller")
```

Comment:
Could we name this log `verbosity` instead of `loglvl`? I feel like it is a bit more indicative of what we're setting with this. Also, it may make sense in the description to add that the higher the number, the less verbose it is.

Reply:
Sounds good to me.